integration-cli/docker_api_inspect_unix_test.go

```diff
@@ -16,7 +16,7 @@ func (s *DockerSuite) TestInspectApiCpusetInConfigPre120(c *check.C) {
 	testRequires(c, cgroupCpuset)
 
 	name := "cpusetinconfig-pre120"
-	dockerCmd(c, "run", "--name", name, "--cpuset", "0-1", "busybox", "true")
+	dockerCmd(c, "run", "--name", name, "--cpuset-cpus", "0-1", "busybox", "true")
```

Comment:
yea, thinking about this more - based on the "#" in the code for "--cpuset" I think you're right.  The typo is the missing "-" in the deprecated.md file.